code/mission/import/xwinglib.cpp

```diff
@@ -72,6 +72,20 @@ struct xwi_flightgroup {
 	short primary_target;
 	short secondary_target;
 };
+
+struct xwi_objectgroup {
```

Comment:
The `xwi_objectgroup` struct must exactly match the file format reference, even if the `XWMObject` struct has more fields or fewer fields.  That's why I called `XWMObject` the "friendly" struct, because it is friendlier to what the user might do with it.  By contrast, you can't add or remove fields to/from `xwi_objectgroup` because the bytes will be misaligned.